db/migrate/20150910170527_add_data_to_credit_card.rb

```diff
@@ -1,5 +1,9 @@
 class AddDataToCreditCard < ActiveRecord::Migration
   def change
-    add_column :spree_credit_cards, :data, :json
+    if ActiveRecord::Base.connection.adapter_name != 'SQLite' && ActiveRecord::Base.connection.supports_json?
```

Comment:
@athal7 looking at the updates of that ```rails``` PR, their idea is to include ```supports_json?``` methods only in rails 5.0 so my suggestion is to remove my second commit to be compliant to the current version of ```ActiveRecord```, what do you think? /cc @kamui